controllers/mainController.js

```diff
@@ -0,0 +1,34 @@
+const Todos = require('../models/todo');
```

Comment:
Please, don't name controller **MainController**. If this controller is related to the todos then call it **TodoController**.